docs/pages/includes/install-linux.mdx

```diff
@@ -41,52 +38,50 @@ Use the appropriate commands for your environment to install your package.
   ```
 
   </TabItem>
-  <TabItem label="Amazon Linux 2/RHEL (RPM)" scope="oss">
+  <TabItem label="AL 2/RHEL 7 (yum)" scope="oss">
 
   ```code
   # Source variables about OS version
   $ source /etc/os-release
   # Add the Teleport YUM repository for v(=teleport.major_version=). You'll need to update this
   # file for each major release of Teleport.
-  # Note: if using a fork of RHEL/CentOS or Amazon Linux you may need to use '$ID_LIKE'
-  # and the codename your distro was forked from instead of '$ID'
-  # Supported versions are listed here: https://github.com/gravitational/teleport/blob/master/build.assets/tooling/cmd/build-os-package-repos/runners.go#L133-L153
-  $ sudo yum-config-manager --add-repo $(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/v(=teleport.major_version=)/teleport.repo")
+  $ sudo yum-config-manager --add-repo "$(rpm --eval "https://yum.releases.teleport.dev/$ID/$VERSION_ID/Teleport/%{_arch}/stable/v(=teleport.major_version=)/teleport.repo")"
   $ sudo yum install teleport
   #
   # Tip: Add /usr/local/bin to path used by sudo (so 'sudo tctl users add' will work as per the docs)
   # echo "Defaults    secure_path = /sbin:/bin:/usr/sbin:/usr/bin:/usr/local/bin" > /etc/sudoers.d/secure_path
-  #
-  # Optional:  Use DNF on newer distributions
-  # $ sudo dnf config-manager --add-repo https://rpm.releases.teleport.dev/teleport.repo
-  # $ sudo dnf install teleport
   ```
-
-  <Admonition type="warning" title="Amazon Linux 2023 Installations">
-  Amazon Linux 2023 users should utilize the tarball or RPM file to install Teleport.
-  You can download the matching RPM architecture for your instance
-  from the [Teleport Downloads page](https://goteleport.com/download) and install with the example command below.
+  </TabItem>
+  <TabItem label="AL 2023/RHEL 8+ (dnf)" scope="oss">
```

Comment:
Keep as Amazon Linux vs AL.  If you search AL linux in duck duck go you get Alpine and Alma Linux as an example.